pages/virtual-leagues/[type].tsx

```diff
@@ -35,11 +52,30 @@ const VirtualLeaguePage: NextPage<PageProps> = ({ res, verifiedType }) => {
 };
 
 VirtualLeaguePage.getInitialProps = async (ctx: NextPageContext) => {
-  const verifiedType = toVerifiedType(ctx.query.type as string);
-  var res = await getVirtualLeagues(verifiedType);
+
+  const verifiedType =  toVerifiedType(ctx.query.type as string);
+
+  if(verifiedType == null){
+
+    if (ctx.res)
```

Comment:
what does this actually check?

Reply:
Null. TS vil ikke la meg gjøre det uten. Vet ikke hvorfor den kan være null her.

Reply:
men vi bruker ikke ctx til noe, så er vel ikke vits å sette statusCode?

Reply:
Hm? Det var bakgrunnen for PRen. Angi 404 statuskode når man prøver nå en ikke-eksisterende virtual league..? Tenkte du på at det er en annen måte å sette statuskode på?

Reply:
sorry jeg er dum, ja det blir så klart riktig 😄